data/json/overmap/overmap_special/specials.json

```diff
@@ -1177,6 +1177,20 @@
     "occurrences": [ 0, 30 ],
     "flags": [ "WILDERNESS" ]
   },
+  {
+    "type": "overmap_special",
+    "id": "Rat Cave",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "cave" },
+      { "point": [ 0, 0, -1 ], "overmap": "cave_rat_underground" },
+      { "point": [ 0, 0, -2 ], "overmap": "cave_rat" }
+    ],
+    "locations": [ "forest" ],
+    "city_sizes": [ 0, 12 ],
+    "city_distance": [ 5, -1 ],
+    "occurrences": [ 0, 1 ],
```

Comment:
Just a note, the description for the PR says:

> Rat cave will now spawn much rarer (50% chance to spawn on each overmap).

But this is not a 50% chance to spawn. This just means it will spawn between 0 and 1 times per overmap, and the actual % chance of success is going to depend on whether this special gets selected and whether it can match criteria.

If you want something more like a 50% chance, change occurrences to `[ 50, 100 ]` and add the "UNIQUE" flag to the special. This will make it so that it has a 50% chance per overmap of being either not allowed at all, or made a mandatory special and thus getting first chance at placement (and thus more likely to be placed if it succeeds its roll).

Reply:
Yeah. It's just incorrect description.